services/blockchain-indexer/shared/database/schema/accounts.js

```diff
@@ -17,21 +17,19 @@ module.exports = {
 	primaryKey: 'address',
 	schema: {
 		address: { type: 'string' },
-		publicKey: { type: 'string' },
-		isGenesisAccount: { type: 'boolean', null: false, defaultValue: false },
+		balance: { type: 'bigInteger', null: false, defaultValue: 0 },
 		isDelegate: { type: 'boolean', null: false, defaultValue: false },
-		balance: { type: 'bigInteger' },
-		username: { type: 'string' },
-		rewards: { type: 'bigInteger', defaultValue: 0 },
-		producedBlocks: { type: 'integer', defaultValue: 0 },
-		totalVotesReceived: { type: 'string' },
+		publicKey: { type: 'string', null: true },
+		name: { type: 'string', null: true },
+		generatorKey: { type: 'string', null: true },
+		blsKey: { type: 'string', null: true },
+		proofOfPosession: { type: 'string', null: true }, // Probably unnecessary
+		rewards: { type: 'bigInteger', null: false, defaultValue: 0 },
+		producedBlocks: { type: 'integer', null: false, defaultValue: 0 },
```

Comment:
Please remove these columns from the `accounts` table and create a new `generators` table with them.